projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -2312,6 +2314,28 @@ static function success_message( $feedback_id, $form ) {
 		);
 	}
 
+	/**
+	 * Returns a script that disables the contact form button after a form submission.
+	 *
+	 * @return string The script.
+	 */
+	private static function get_script_for_form() {
+		return "<script>
+			( function () {
+				const contact_forms = document.getElementsByClassName('contact-form');
+				const buttons = document.getElementsByTagName('button');
```

Comment:
You're fetching all the buttons in the page here, not just the ones that belong to the form. Is that intended?

Reply:
Just to close the loop on this comment, I decided to disable only the submitted form's button elements in commit 008d53c. This means that the visitor could, in theory, still do things like submit multiple forms in the same page reload if the page has multiple forms. I think that behavior is acceptable. The same form being submitted multiple times is not acceptable behavior, and this PR prevents that from happening.